docs/samples/cabundle/README.md

```diff
@@ -0,0 +1,100 @@
+# KServe with Self Signed Certificate Model Registry
+
+If you are using a model registry with a self-signed certificate, you must either skip ssl verify or apply the appropriate cabundle to the storage-initializer to create a connection with the registry.
+This document explains three methods that can be used in KServe.
+
+- Configure CaBundle for storage-initializer
+  - Global configuration
+  - Using `storage-config` Secret
+
+- Skip SSL Verification
+  
+## Configure CaBundle for storage-initializer  
+### Global Configuration
+
+KServe use `inferenceservice-config` ConfigMap for default configuration. If you want to add `cabundle` cert for every inferenceservice, you can set `caBundleSecretName` in the ConfigMap. Before updating the ConfigMap, you have to create a secret for cabundle certificate in the namespace that KServe controller is running and the data key in the Secret must be cabundle.crt. 
+
+- Create a Secret with the cabundle cert
+  ~~~
+  kubectl create secret generic cabundle --from-file=/path/to/cabundle.crt
+
+  kubectl get secret cabundle -o yaml
+  apiVersion: v1
+  data:
+    cabundle.crt: LSXXXK
+  kind: Secret
+  metadata:
+    name: cabundle
+    namespace: kserve
+  type: Opaque
+  ~~~
+- Update `inferenceservice-config` ConfigMap 
+  ~~~
+    storageInitializer: |-
+    {
+        ...
+        "caBundleSecretName": "cabundle",
+        ...
+    }
+  ~~~
+  
+If you update this configuration after, please restart KServe controller pod.  
+
+### Using storage-config Secret
```

Comment:
Would this support the [common storage secret](https://github.com/kserve/kserve/tree/master/docs/samples/storage/storageSpec#create-common-storage-secret) format?

If so, for the common storage we need either (1) a way to attach multiple bundles to the single secret, or (2) be able to specify a bundle for each entry in the single secret. But that is not mentioned here.

Reply:
No, it is a special option so annotation would be enough.